mlflow/pytorch/_lightning_autolog.py

```diff
@@ -287,6 +292,174 @@ def on_test_end(self, trainer, pl_module):
         self.metrics_logger.flush()
 
 
+LATEST_CHECKPOINT_ARTIFACT_TAG_KEY = "_latest_checkpoint_artifact"
```

Comment:
This might annoy users when it shows up on MLflow UI.

Reply:
What about changing the tag key to "mlflow.latest_checkpoint_artifact` so that UI would hide it ?

Reply:
can we move the tag in mlflow/utils/mlflow_tags.py?